### DISCUSSION TITLE
appchannel_test not working

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#233] [Q&A]
I am trying to run the "app_appchannel_test" example, found in the Bitcraze VM at /home/bitcraze/Desktop/projects/crazyflie-firmware/examples/

I have Crazyflie 2.0, and the radio. The radio is plugged in, and the Crazylie set to bootloader mode. I successly ran "make && make cload". This runs successfully and Crazyflie starts up.

But when I type "appchannelTest.py", this is what I get:
-----------------------------------
Scanning interfaces for Crazyflies...
Crazyflies found:
No Crazyflies found, cannot run example
-----------------------------------
Please advise.

Also, is "make" a synonym for "compile"? Thanks!

**Comment by knmcguire** [comment]
Hi!

If you connect to the same crazyflie with the cfclient and look at the consoletab, can you see the [print out ](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/console_tab/)of the app layer printing at all?
